RockPaperScissors/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace RockPaperScissors
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Player player1 = new Player(0);
+            Player player2 = new Player(0);
+
+            List<string> whoWon = new List<string>();
+            whoWon.Add("Both players win.");
+            whoWon.Add("Player 1 wins.");
+            whoWon.Add("Player 2 wins.");
+
+            for(int i = 0; i < 10; i++)
+            {
+                string p1 = player1.GetMove();
+                string p2 = player2.GetMove();
+
+                string statement = "Player 1 played " + p1 + " and player 2 played " + p2 + ".";
+
+                if (p1 == p2)
+                {
+                    player1.numberOfRoundsWon++;
+                    player2.numberOfRoundsWon++;
+                    Console.WriteLine(statement + whoWon[0]);
```

Comment:
Make 0, 1, 2 `const` values with names so we know which win conditions they represent